chart/values.yaml

```diff
@@ -3,8 +3,12 @@ image:
   registry: docker.io
   # -- Image registry's namespace
   repo: openebs
-  # -- Release tag for our images
-  tag: develop
+  tags:
+    # -- Release tag for our images
+    tag: develop
+    controlTag:
+    dataTag:
+    extTag:
```

Comment:
```suggestion
    tag: develop
    # Note: Below image tag configuration is optional and typically should never be used. Setting
    #           specific image tags for the different repositories proves useful for some integration testing
    #           scenarios. Use the 'tag' option above to set released/pre-released container image tags.
    # Set image tag for control-plane components
    control-plane: ""
    # Set image tag for data-plane components
    data-plane: ""
    # Set image tag for extensions components
    extensions: ""
```

Reply:
Should `tags.tag` be `tags.global` so it's clearer? Or maybe keep image.tag?

Reply:
image.tag would be better. Along with comments as provided by @niladrih  above, I will make relevant changes and resend for review

Reply:
Incorporated these changes